modules/ppcp-button/resources/js/modules/Helper/ScriptLoading.js

```diff
@@ -87,3 +87,11 @@ export const loadPaypalJsScript = (options, buttons, container) => {
         paypal.Buttons(buttons).render(container);
     });
 }
+
+export const loadPaypalJsScriptPromise = (options) => {
```

Comment:
:thinking: 

why not use the `loadPaypalScriptPromise` above?

Reply:
@AlexP11223 I need to reload the script with different properties (ex. `components: 'card-fields'` for ACDC) each time customer switch from PayPal to ACDC payments, it seems `loadPaypalScriptPromise` does not allow to do so: https://github.com/woocommerce/woocommerce-paypal-payments/blob/trunk/modules/ppcp-button/resources/js/modules/Helper/ScriptLoading.js#L16